packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -191,62 +207,84 @@ export class HbarLimitService implements IHbarLimitService {
 
     const dailyLimit = HbarLimitService.DAILY_LIMITS[spendingPlan.subscriptionType];
 
+    // note: estimatedTxFee is only applicable in a few cases (currently, only for file transactions).
+    //      In most situations, estimatedTxFee is set to 0 (i.e., not considered).
+    //      In such cases, it should still be true if spendingPlan.spentToday === dailyLimit.
     const exceedsLimit = spendingPlan.spentToday >= dailyLimit || spendingPlan.spentToday + estimatedTxFee > dailyLimit;
-    this.logger.trace(
-      `${requestIdPrefix} ${user} ${exceedsLimit ? 'should' : 'should not'} be limited: spentToday=${
-        spendingPlan.spentToday
-      }, estimatedTxFee=${estimatedTxFee}, dailyLimit=${dailyLimit}`,
-    );
+
+    if (exceedsLimit) {
+      this.logger.warn(
+        `${requestIdPrefix} User has exceeded HBAR rate limit threshold: user=${user}, spentToday=${spendingPlan.spentToday}, estimatedTxFee=${estimatedTxFee}, dailyLimit=${dailyLimit}, txConstructorName=${txConstructorName}, mode=${mode}, methodName=${methodName}`,
+      );
```

Comment:
This update offers an alternative way to express this log. The previous implementation would issue a log at this position regardless; therefore, if we are affraid that "it would only create spam in the logs," we should consider removing this entire log instead.

However, as a developer who traverses logs, all the time, for debugging, I believe that having more generic logs is beneficial.

Reply:
In this case, let’s change the level of the log to `trace` instead of `warning`

Reply:
I think this kind of information makes more sense in a metric, but keeping it and setting it to trace is a good idea for now.

Reply:
Fair point. Updated.